hack/install-velad.sh

```diff
@@ -37,7 +37,7 @@ getSystemInfo() {
 }
 
 verifySupported() {
-    local supported=(darwin-amd64 darwin-arm64 linux-amd64 linux-arm linux-arm64)
+    local supported=(darwin-amd64 linux-amd64)
```

Comment:
why don't keep linux-arm64 since you have supported

Reply:
I miss this, I can split this [PR](https://github.com/kubevela/velad/pull/31)